action/messageHubProduce.py

```diff
@@ -108,6 +105,42 @@ def main(params):
 
     return result
 
+def produceBatchedMessages(producer, topic, messageKey, values):
+    logging.info("Producing batched messages")
+
+    if messageKey is not None:
+        for val in values:
+            producer.send(topic, bytes(val, 'utf-8'), key=bytes(messageKey, 'utf-8'))
+    else:
+        for val in values:
+            producer.send(topic, bytes(val, 'utf-8'))
+
+    producer.flush()
```

Comment:
Cannot get partition and offset now for log statement?

Reply:
Doc for `flush()`: http://kafka-python.readthedocs.io/en/master/apidoc/KafkaProducer.html#kafka.KafkaProducer.flush.